src/lib/protocols/viber.c

```diff
@@ -42,14 +42,6 @@ static void ndpi_search_viber(struct ndpi_detection_module_struct *ndpi_struct,
   
   NDPI_LOG_DBG(ndpi_struct, "search for Viber\n");
 
-  if(packet->udp && packet->iph) {
-    /* ignore broadcast as this isn't viber */
-    if((packet->iph->saddr == 0xFFFFFFFF) || (packet->iph->daddr == 0xFFFFFFFF)) {
-      NDPI_EXCLUDE_PROTO(ndpi_struct, flow);
-      return;
-    }
-  }
-
```

Comment:
Can you explain why removing this block?

Reply:
It should have been deleted by mistake